_data/simple-icons.json

```
@@ -6738,6 +6738,11 @@
             "hex": "F06B66",
             "source": "https://mailgun.com"
         },
+        {
+            "title": "Mailtrap",
+            "hex": "22D172",
+            "source": "https://mailtrap.io/wp-content/uploads/2022/12/mailtrap-new-logo.svg"
```

I would alter the source to just be the website. If they ever change the filename, we'll lose this source.

```suggestion
            "source": "https://mailtrap.io"
```